src/App.tsx

```diff
@@ -11,23 +11,26 @@ const App: React.FC = () => {
   const [todos, setTodos] = useState<Todo[]>([]);
   const [loading, setLoading] = useState(false);
   const [isLoaded, setIsLoaded] = useState(false);
+  const [isError, setIsError] = useState(false);
 
   const startLoadingFromServer = async () => {
     setLoading(true);
 
-    const todosFromServer = await getTodos();
-    const usersFromServer = await getUsers();
+    try {
+      const todosFromServer = await getTodos();
+      const usersFromServer = await getUsers();
 
-    const preparedTodos = todosFromServer.map(todo => ({
-      ...todo,
-      user: usersFromServer.find(user => user.id === todo.userId),
-    }));
+      const preparedTodos = todosFromServer.map(todo => ({
+        ...todo,
+        user: usersFromServer.find(user => user.id === todo.userId),
+      }));
 
-    setTimeout(() => {
       setInitialTodos(preparedTodos);
       setTodos(preparedTodos);
       setIsLoaded(true);
-    }, 500);
+    } catch (error) {
+      setIsError(true);
```

Comment:
```suggestion
      setIsError(error);
```
желательно еще где-то выводить что именно пошло не так